cluster/pulumi/infra/grafana-dashboards/canton-network/catchup.json

```diff
@@ -145,7 +145,7 @@
           "disableTextWrap": false,
           "editorMode": "code",
           "exemplar": false,
-          "expr": "delta(daml_sequencer_block_delay{namespace=~\"$namespace\", component=\"sequencer\",job=~\"global-domain-$migration-sequencer\"}[5m]) / 5",
+          "expr": "max by (namespace, job) (delta(daml_sequencer_block_delay{namespace=~\"$namespace\", component=\"sequencer\",job=~\"global-domain-$migration-sequencer\"}[5m])) / 5",
```

Comment:
neither min nor max makes sense here. Ideally we would drop the namespace and job label at each point and then take the delta across that. But you can't do that. You need to use the delta first and then aggregate. This seems good enough™.